src/Experimental/Traversals/bfs.jl

```diff
@@ -1,4 +1,5 @@
-import Base.Sort, Base.Sort.Algorithm
+using Base.Sort: Sort
+using Base.Sort.Algorithm: Algorithm
```

Comment:
```suggestion
import Base.Sort
import Base.Sort.Algorithm
```

Reformatting broke these imports